src/commands.rs

```diff
@@ -813,6 +813,18 @@ implement_commands! {
         cmd("ZPOPMIN").arg(key).arg(count)
     }
 
+    /// Removes and returns up to count members with the highest scores, 
+    /// from the first non-empty sorted set in the provided list of key names.
+    fn zmpopmax<K: ToRedisArgs>(keys: &'a [K], count: isize) {
```

Comment:
Naming convention: `zmpop_max`

Reply:
Are you sure this works as expected? I don't see `COUNT` specified

Reply:
Sorry, perhaps I'm misinterpreting here, but shouldn't the command at a minimum look like the following:

```cmd("ZMPOP").arg(keys.len()).arg(keys).arg("MAX").arg("COUNT").arg(count)```

I don't believe `count` can be specified without a preceding `COUNT` arg

Reply:
Oh, I missed the part about the explicit `COUNT`.